libs/client/common/stores/src/lib/park.store.ts

```diff
@@ -47,7 +48,7 @@ export class ParkStore implements Store {
   async getParkWithExercise(
     parkId: Park['id'],
     exerciseId: Exercise['id']
-  ): Promise<(Park & { exercises: [Exercise] }) | undefined> {
+  ): Promise<Park | undefined> {
```

Comment:
:( Warum hosch des ussa to haha des mar a mega coole line ma sia direkt dasses a array zruck gibt mir genau einer exercise